src/main/java/wooteco/subway/member/domain/Member.java

```diff
@@ -28,6 +28,10 @@ public Member(String email, String password, Integer age) {
         this.age = age;
     }
 
+    public Boolean isSamePassword(String password) {
+        return this.password.equals(password);
+    }
```

Comment:
```suggestion
    public boolean isSamePassword(String password) {
        return this.password.equals(password);
    }
```

어떤 차이가 있을까요? 
`이펙티브 자바 item 61. 박싱된 기본 타입보다는 기본 타입을 사용하라` 를 읽어 보면 좋을 것 같아요.